RELEASE-NOTES.md

```diff
@@ -1,3 +1,59 @@
+# Riak CS 1.5.4 Release Notes
+
+## Bugs Fixed
+
+- Disable previous riak object after backpressure sleep is triggered
+  [riak_cs/#1041](https://github.com/basho/riak_cs/pull/1041). This
+  change prevents unnecessary siblings increase when backpressure is
+  triggered under high upload concurrency, and uploads are interleaved
+  during the backpressure sleep.
+- Fix wrong path rewrite in S3 API by unnecessary unquote
+  [riak_cs/#1040](https://github.com/basho/riak_cs/pull/1040). For
+  example, in case when '+' is included in path, '+' (plus, `%2B` in
+  quoted representation) in object key has been mistakenly unquoted to
+  become ' ' (whitespace, `%20` in quoted representaiton). This fix
```

Comment:
I think we usually use Unicode as character set these days, the name of U+0020 is just "space" [1].
And the word "whitespace" is used for broader characters [2].

So, very tiny nit, I prefer the word "space". 

[1] http://www.fileformat.info/info/unicode/char/0020/index.htm
[2] http://en.wikipedia.org/wiki/Whitespace_character#Unicode